tests/conftest.py

```
@@ -0,0 +1,112 @@
+        print(f"MinIO not ready yet: {str(e)}")
+        raise
+
+
+def wait_for_services():
+    """Wait for all required services to be ready."""
+    try:
+        wait_for_api()
+        wait_for_minio()
+        # Add more service checks as needed
+    except Exception as e:
+        raise Exception(f"Services failed to start: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)